src/static/javascripts/components/dashboard.jsx

```diff
@@ -93,19 +97,22 @@ var Page = React.createClass({
     }.bind(this)
     });
     },
-    */
+    //This is where the initial loadPageJSON call happens, it happens when the React class is instantiated (Carlos we should put that 
+    //initial fetch Sam showed you here)
     getInitialState: function() {
+    this.loadPageJSON();
```

Comment:
Carlos you should put your initial data fetch here, right below I think we can change componentDidMount to something else and be set for future fetches of data.